core/trino-main/src/test/java/io/trino/execution/scheduler/TestUniformNodeSelector.java

```diff
@@ -137,8 +137,6 @@ public void testQueueSizeAdjustmentScaleDown()
                 false,
                 queueSizeAdjuster);
 
-        Set<Split> splits = new LinkedHashSet<>();
```

Comment:
Undo

Reply:
This test already (before this patch) mutates the following class fields: `nodeTaskMap`, `nodeManager`, `nodeSelector`, and `taskMap`.